plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DataFileInfo.java

```diff
@@ -13,79 +13,29 @@
  */
 package io.trino.plugin.deltalake;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import io.trino.plugin.deltalake.transactionlog.statistics.DeltaLakeJsonFileStatistics;
 
 import java.util.List;
 
 import static java.util.Objects.requireNonNull;
 
-public class DataFileInfo
+public record DataFileInfo(
+        String path,
+        long size,
+        long creationTime,
+        io.trino.plugin.deltalake.DataFileInfo.DataFileType dataFileType,
```

Comment:
json field name will change. It was `fileType` before.

Not sure if this matters since this doesn't seem to be persisted.

Reply:
in a few other records too but none of them seem to be persisted.